spec/spec_helper.rb

```diff
@@ -1,8 +1,62 @@
-require 'coveralls'
-require 'simplecov'
+ENV['RACK_ENV'] = 'test'
```

Comment:
Not sure what happened.  Maybe I deleted them by mistake and added them back a commit later.  Anyway, yes, I understand they are required.